src/actions/LogActions.tsx

```diff
@@ -28,6 +32,9 @@ interface LogOutput {
   loggedInUser?: LoggedInUser
   data: string
 }
+export interface LogOutputOptions {
+  userMessage: string
+}
```

Comment:
Unused now?

Reply:
Yup, removed along with the redundant `textStyle` and associated `<View>...</View>`